### PR TITLE
[FUSETOOLS2-1644] Updated developers info. 

### PR DESCRIPTION
**Pull request by joshiraez**
Left Lars as a contributtor this time as he still contributed to the codebase and it made sense

But if it was a mistake feel free to comment/edit :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,11 @@
 			"name": "Lars Heinemann",
 			"email": "lheinema@redhat.com",
 			"url": "http://redhat.com"
+		},
+		{
+			"name": "Joshi Raez",
+			"email": "jraez@redhat.com",
+			"url": "http://redhat.com"
```
Comment (apupier) [inline package.json:32]: I noticed that above it is not done but it is better to use https url everywhere, it avoids potential security issues

**Review by apupier — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (joshiraez) [reply package.json]: Fixed

**Comment by apupier** [comment]
Can you rebase please? The issue on Mac should have been fixed

**Comment by joshiraez** [comment]
I think the MacOS tests are still broken. I canceled the job because it had been running for 50m and there was an error "Unable to retrieve AtlasMap frame" over and over.

**Review by apupier — APPROVED** [review]

**Pull request merged**